server/fleet/apple_mdm.go

```diff
@@ -432,6 +432,18 @@ func (p MDMAppleSettingsPayload) AuthzType() string {
 	return "mdm_apple_settings"
 }
 
+// MDMAppleSetupPayload describes the payload accepted by the endpoint to
+// update specific MDM macos setup values for a team (or no team).
+type MDMAppleSetupPayload struct {
+	TeamID                      *uint `json:"team_id"`
+	EnableEndUserAuthentication *bool `json:"enable_end_user_authentication"`
+}
+
+// AuthzType implements authz.AuthzTyper.
+func (p MDMAppleSetupPayload) AuthzType() string {
+	return "mdm_apple_settings" // TODO: add mdm_apple_setup to rego?
```

Comment:
maybe use a name that applies to both? IIRC they almost were part of the same endpoint.